src/components/Restoration/YourCompanySummary.vue

```diff
@@ -26,13 +26,13 @@
             <div>[TODO - Applicant's relationship: Director, Shareholder]</div>
           </v-col>
         </v-row>
-        <v-row no-gutters class="mt-3" v-if="hasFileNumber">
+        <v-row no-gutters class="mt-3" v-if="getStateFilingRestoration.approvalType === ApprovalTypes.VIA_COURT_ORDER">
           <v-col cols="3">
             <label><strong>Approval Type</strong></label>
           </v-col>
           <v-col cols="8" class="mt-n1">
             <div class="font-weight-bold">Approved by Court Order</div>
-            <div>Court Order Number: {{ getFileNumber }}</div>
+            <div v-if="getRestoration.courtOrder">Court Order Number: {{ getRestoration.courtOrder.fileNumber }}</div>
```

Comment:
I'd be tempted to use a couple of local computed methods (getters) to replace the code here and on line 29, just to keep the template super clean.

`isApprovedByCourtOrder` and `courtOrder`, if there aren't already getters for them.